peer/brontide.go

```diff
@@ -489,6 +508,16 @@ type Brontide struct {
 
 	// log is a peer-specific logging instance.
 	log btclog.Logger
+
+	// SendBackup indicates if to send back up to this field.
+	SendBackup int
+
+	// LatestPeerBackup is the latest backup we have stored with a peer.
+	LatestPeerBackup []byte
+
+	// SendBackupMtx provides a concurrency safe way to update the
+	// `sendBackup` field as well as `LatestPeerBackup` field.
+	SendBackupMtx sync.RWMutex
```

Comment:
These fields need a lot more explanation. This commit seems split between whether or not it is implementing the request layer for storing our data on our peer using this protocol, or if it is implementing the storage layer locally.

Reply:
What do you not understand about the fields.